docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -661,7 +661,7 @@ export const STYLES_APIBOX = css({
   },
 
   [`@media screen and (max-width: ${breakpoints.medium + 124}px)`]: {
-    padding: `0 ${spacing[4]}px`,
+    paddingInline: `${spacing[4]}px`,
```

Comment:
Nice catch, thank you for the correction! 👍 

```suggestion
    paddingInline: spacing[4],
```

Small simplification, since for single value we can pass number directly.

Reply:
It is my pleasure, 
thanks for the hint, pushed it in latest commit